src/js/constants/configConstants.js

```diff
@@ -5,3 +5,10 @@ export const publicOverviewUrl = '/overview';
 export const searchFieldsUrl = '/fields';
 export const katsuUrl = '/katsu';
 export const provenanceUrl = '/provenance';
+
+export const translationEnUrl = '/public/locales/en/translation_en.json';
+export const translationFrUrl = '/public/locales/fr/translation_fr.json';
```

Comment:
These both look like dead code to me. Isn't this handled by the 118n `backend.loadpath`?

Reply:
ooops yes